contracts/bondingcurve/CommonsToken.sol

```diff
@@ -40,6 +40,7 @@ contract CommonsToken is BondingCurveToken, Pausable {
   uint256 public theta; // fraction (in PPM) of the contributed amount that goes to the funding pool
   uint256 public p0; // price (in externalToken) for which people can purchase the internal token during the hathing phase
   uint256 public initialRaise; // the amount of external tokens that must be contributed during the hatching phase to go post-hatching
+  uint256 private initialRaiseInternal; // the amount of INTERNAL tokens that are minted during the hatching phase.
```

Comment:
Any reason you want to keep this private? If it would be public we could display this information in the UI as part of a current hatching progress.

Reply:
OK. I don't any major reasons why it can't be made public. The only thing is the name would probably need to change? 'Internal' is a naming convention of the internal business logic.